app/src/components/Map/ControlPanel.jsx

```diff
@@ -32,30 +32,52 @@ class ControlPanel extends Component {
     }
   }
 
+  calculateVesselCount() {
+    const min = this.props.timelineInnerExtent[0].getTime();
+    const max = this.props.timelineInnerExtent[1].getTime();
+
+    if (this.props.chartData.length === 0) {
+      return '-';
+    }
+
+    let result = this.props.chartData.reduce((acc, elem) => {
+      if (elem.date >= min && elem.date <= max) {
+        return acc + elem.value;
+      }
+      return acc;
+    }, 0);
+
+    result = `${Math.round(result)}`;
+    for (let i = result.length - 3; i > 0; i -= 3) {
```

Comment:
Please replace by something more concise:
```
result.toLocaleString()
```
Performance is not good on 1000s of elemements, but in that case with one element the overhead is neglectible.